notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>Remove "Get signatures" heading</p><p> </p><p>You can get the signatures of your model's input and output layers by reloading the model into memory, and querying it for the signatures corresponding to each layer.</p><p> </p><p>You'll need the signatures for the following layers:</p><p> </p><ul><li>Serving Function input layer</li><li>Model input layer</li><li>Model output layer</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>